hack/ebs-scale-test/helpers/cluster-setup/manage-cluster.sh

```diff
@@ -72,6 +73,73 @@ deploy_ebs_csi_driver() {
     "$path_to_chart"
 }
 
+## Karpenter
+
+deploy_karpenter() {
+  echo "Deploying Karpenter Pre-requisites"
+  aws eks update-kubeconfig --name "$CLUSTER_NAME" --region "$AWS_REGION"
```

Comment:
Suggestion: Remove this line since `aws eks update-kubeconfig` is already getting called in `create_cluster`, which is executed right before this `deploy_karpenter` function.